projects/immutablex/index.js

```diff
@@ -0,0 +1,17 @@
+const ADDRESSES = require('../helper/coreAssets.json');
+const { sumTokensExport } = require('../helper/sumTokens');
+
+const owners = [
+  '0x5FDCCA53617f4d2b9134B29090C87D01058e27e9',
+];
+
+const tokens = [
+  ADDRESSES.null,
+  ADDRESSES.ethereum.OMI,
```

Comment:
arent these two their own tokens?

Reply:
I don't think any of these are their own tokens. OMI is the currency of [Ecomi](https://www.ecomi.com/) which is a mobile platform that has [partnered with Immutable](https://medium.com/ecomi/veve-moves-to-ethereum-via-immutable-x-d69bfbd736d6) and GODS is the currency of Gods Unchained who [have launched on Immutable](https://blog.godsunchained.com/2021/03/04/immutable-x-gu-rollout/).